anvio/tests/unit/test_homogeneityindex.py

```diff
@@ -214,13 +220,14 @@ def setUp(self):
         self.quick_geometric_similarity = sum([aln_col1_similarity_score,
                                                aln_col2_similarity_score,
                                                aln_col3_similarity_score,
-                                               aln_col4_similarity_score]) / num_genes
```

Comment:
dividing by `num_genes` should really be dividing by the number of residues (aka alignment columns).  The commit fixes this.

Reply:
(The way the test was set up originally, number of genes was equal to number of residues, so it didn't actually cause a problem, it was just confusing.

Reply:
always something tricky hides the most important bugs :)